.github/workflows/stale_issues.yml

```diff
@@ -0,0 +1,45 @@
+name: Close Stale Issues
+
+on:
+  # Runs daily at 10:00 AM UTC
+  schedule:
+    - cron: '0 10 * * *'
+  # Runs at issue comment
+  issue_comment:
+    types: [created]
+
+  # Allow manual trigger
+  workflow_dispatch:
+
+jobs:
+  manage-stale-issues:
+    runs-on: ubuntu-latest
```

Comment:
Can you update this to run on `ubuntu-24.04`? I like to use specific lts versions